vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1014,7 +999,9 @@ private void updateSelectedFragment(final MenuItem item) {
                     fragment = PeopleFragment.newInstance();
                 }
                 mCurrentFragmentTag = TAG_FRAGMENT_PEOPLE;
-                mSearchView.setQueryHint(getString(R.string.home_filter_placeholder_people));
+                if(getResources().getBoolean(R.bool.enable_riot_search_view)) {
+                    mSearchView.setQueryHint(getString(R.string.home_filter_placeholder_people));
```

Comment:
Just double checking that you intentionally don't have:
     mSearchView.setVisibility(View.VISIBLE);
in this if statement

Reply:
visibility is not actually needed anymore as Lingo has a separate searchview. deleted those codes.

Reply:
Can you please explain more the statement "Lingo has a separate searchview".  So is mSearchView the Lingo Search View or the Search View for something else (I'm not sure what this something else is)?  I suggest that code comments describing what each variable represents will be required for clarity.

Reply:
I am just using the same variable mSearchView for Lingo searchview as well as Riot searchview.